src/plugins/intel_gpu/include/intel_gpu/runtime/options.inl

```diff
@@ -46,6 +46,8 @@ OV_CONFIG_RELEASE_INTERNAL_OPTION(ov::intel_gpu, allow_new_shape_infer, false, "
 OV_CONFIG_RELEASE_INTERNAL_OPTION(ov::intel_gpu, use_onednn, false, "Enable/Disable onednn for usage for particular model/platform")
 OV_CONFIG_RELEASE_INTERNAL_OPTION(ov::intel_gpu, max_kernels_per_batch, 8, "Controls how many kernels we combine into batch for more efficient ocl compilation")
 OV_CONFIG_RELEASE_INTERNAL_OPTION(ov::intel_gpu, impls_cache_capacity, 300, "Controls capacity of LRU implementations cache that is created for each program object for dynamic models")
+OV_CONFIG_RELEASE_INTERNAL_OPTION(ov::intel_gpu, asym_dynamic_quantization, false, "Enforce asymmetric mode for dynamically quantized activations")
+OV_CONFIG_RELEASE_INTERNAL_OPTION(ov::intel_gpu, dynamic_quantization_all, false, "Enforce dynamic quantization for all inputs");
```

Comment:
IMO, flag meaning is not clear with this name. If the intent is to disable optimization for 2nd token in runtime and enforce dynamic quantization for all input shapes, then you can probably align naming with other knobs that control runtime optimizations, i.e. something like `disable_dynamic_quantize_opt`
And the option should have DEBUG visibility I suppose